src/views/virtualmachines/details/tabs/disk/tables/disk/DiskRow.tsx

```diff
@@ -23,13 +23,13 @@ const DiskRow: React.FC<
   return (
     <>
       <TableData id="name" activeColumnIDs={activeColumnIDs}>
-        <Split hasGutter>
+        <Split className="disk-row-split">
```

Comment:
This should work nicely :) I want to suggest another possible solution: 
you could try replacing the `Split` component with `Stack` component from patternfly https://www.patternfly.org/v4/layouts/stack which I think will have a cleaner solution to this issue

Reply:
@avivtur please re-review